elsevier/_skip.txt

```diff
@@ -16,9 +15,29 @@ aasri-procedia
 journal-of-adolescent-health
 materials-discovery-today
 pain
-cell
 journal-of-investigative-dermatology
 medical-journal-of-australia
 
 # Moved to Springer
 journal-of-ultrasound
+
+#Cell Press journals
+biophysical-journal
+cancer-cell
+cell
+cell-chemical-biology
+cell-host-and-microbe
+cell-metabolism
+cell-reports
+cell-stem-cell
+current-biology
+developmental-cell
+immunity
+molecular-cell
+neuron
+stem-cell-reports
+structure
+#Cell but different style
+the-american-journal-of-human-genetics
+#Cell, different style, not yet done
+molecular-plant
```

Comment:
@damnation333, if you're up for it, we still need a custom style for the Cell journal Molecular Plant (http://www.cell.com/molecular-plant/home).

@adam3smith, do you have any notes on what is different in this style compared to that of the other Cell journals?